modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheIoManager.java

```diff
@@ -276,11 +277,17 @@ else if (desc.receivedFromStartVersion() != null)
                             }
                         };
 
-                        if (stripe >= 0)
-                            cctx.kernalContext().getStripedExecutorService().execute(stripe, c);
+                        GridKernalContext kernalContext = cctx.kernalContext();
```

Comment:
we usually use ctx for kernal context and cctx for cache context. Please rename kernalContext to ctx

Reply:
Fixed